### ISSUE TITLE
Can't use the extension on Windows

### ISSUE THREAD

**Issue by mameen** [tomrijndorp/vscode-finditfaster#2]
I get this error when I try to use the extension on windows.. I do have fzf, ripgrep, and bat installed and added to the PATH

```
Failed to run checks before starting extension. Maybe this is helpful: Error: Command failed: C:\WINDOWS\system32\cmd.exe /d /s /c "c:\Users\####\.vscode\extensions\tomrijndorp.find-it-faster-0.0.6\flight_check.sh"
```

**Comment by tomrijndorp** [comment]
Hi mameen, thanks for taking the time to check it out and reporting!

On the extension page, I mention:
> **Not tested on Windows**. Likely won't work. I don't know enough about WSL to tell you whether this could work. If you want to contribute, this would be an awesome place to do it.

After you created this ticket, I've tried to figure out exactly what the Windows situation is. Here's what I've found:

**Using WSL inside VS Code, the extension works as expected**. What I'm referring to here is effectively running the VS Code server in Linux as described [here](https://code.visualstudio.com/docs/remote/wsl). In this case, when I have `fzf`, `rg`, and `bat` on my `PATH`, things just work. I'm not a Windows developer myself, and I don't know is this is generally a viable solution. I _think_ it's probably something a lot of Windows users do. If this works for you, great! 🎉 

**Without WSL, loading the extension fails** in the way you described. The reason for this is that the entire extension revolves around a number of (bash) shell scripts, that Windows doesn't quite understand. I'm not a Windows user or developer myself, so I'd have to look more deeply into what can be done.

**What's next**:
1. The error message you received is not very helpful. Fixing this is my top priority and I created #3 for this.
2. Support Windows outside of WSL. The way I see this, there are two options:
    1. Invoke the bash scripts from Windows. I'm not sure how / whether this can be done. Will require WSL to be installed. Not sure if there are further issues with access permissions etc. Will require some investigation.
    2. Create Windows versions of `find_files.sh` and `find_within_files.sh` (the scripts in the extension that invoke `fzf`). This may be the nicest solution as it's a native implementation, but requires Windows shell scripting, which I'm not familiar at all.

To understand how to prioritize 2. and what option to go with, I've created #4. I have no idea how many people use VS Code on Windows (without WSL). My guess is that it's a relatively small group, but I might be mistaken. I'd definitely also be supportive of somebody wanting to implement solution 2.2, which I've also indicated in that ticket.
